packages/mdl-ripple/mdl-ripple.scss

```diff
@@ -26,14 +27,14 @@
   &__foreground,
   &__foreground-circle,
   &__background {
-    width: 100%;
-    height: 100%;
     position: absolute;
+    height: 100%;
+    width: 100%;
   }
 }
 
 .md-ripple__foreground {
-  &-active {
+  &-bounded-active {
   transition: transform 300ms cubic-bezier(0.157, 0.72, 0.386, 0.987);
```

Comment:
If these are modifier classes, I believe the convention is to use two dashes between the base class name and the modifier, e.g. `&--bounded-active`.

Furthermore is there a reason to have compound modifier classes such as `bounded-active` rather than just having two distinct modifier classes? e.g. `&--bounded#{&}--active` ?